tests/browser/test/basic_playground_test.js

```diff
@@ -127,3 +130,99 @@ suite('Right Clicking on Blocks', function (done) {
     await browser.deleteSession();
   });
 });
+
+suite('Disabling', function () {
+  // Setting timeout to unlimited as the webdriver takes a longer
+  // time to run than most mocha tests.
+  this.timeout(0);
+
+  suiteSetup(async function () {
+    browser = await testSetup(testFileLocations.playground);
```

Comment:
This will break when merged to develop due to a refactor of `testFileLocations`.  Merge develop into this PR then fix:
```suggestion
    browser = await testSetup(testFileLocations.PLAYGROUND);
```

Reply:
I will rebase and fix!